src/applications/hca/actions.js

```diff
@@ -9,14 +8,26 @@ export const SUBMIT_ID_FORM_FAILED = 'SUBMIT_ID_FORM_FAILED';
 export function submitIDForm(formData) {
   return dispatch => {
     dispatch({ type: SUBMIT_ID_FORM_STARTED });
-
-    // We need bypass the local API when running locally, so hit dev-api
-    // directly. As an aside, if we want to get a 500 server error when testing
-    // locally, reverse this check to make the app hit the local API instead.
-    const urlPrefix = environment.isLocalhost()
-      ? 'https://dev-api.va.gov/v0'
-      : '';
-    const baseUrl = `${urlPrefix}/health_care_applications/enrollment_status`;
+    /*
```

Comment:
The `enrollment_status` endpoint no longer simply returns 500s when running locally so we don't need to hit `dev-api.va.gov` unless we really want to.